hazelcast/src/test/java/com/hazelcast/jet/core/JobTest.java

```diff
@@ -787,6 +791,63 @@ public void when_suspendedNamedJob_then_newJobWithEqualNameFails() {
                 .isInstanceOf(JobAlreadyExistsException.class);
     }
 
+    @Test
+    public void given_suspensionIsForbidden_when_suspendJob_then_jobIsTerminated() {
```

Comment:
the test method name ("jobIsTerminated") does not match the test behavior

Reply:
Renamed

Reply:
still does not match. `suspend` invocation in the test method does not cancel the job, `cancelAndJoin` does but this is a cleanup method not a method we intend to test

Reply:
Test was updated